kasa/smart/modules/led.py

```diff
@@ -14,7 +14,7 @@ class Led(SmartModule, LedInterface):
 
     def query(self) -> dict:
         """Query to execute during the update cycle."""
-        return {self.QUERY_GETTER_NAME: {"led_rule": None}}
+        return {self.QUERY_GETTER_NAME: {}}
```

Comment:
This change could be done in its own PR, as it might change behavior on some devices?

Reply:
Done